sensor/common/scan/scan.go

```diff
@@ -277,14 +278,15 @@ func (s *LocalScan) getRegistries(ctx context.Context, namespace string, imgName
 
 func (s *LocalScan) getPullSources(srcImage *storage.ContainerImage) []*storage.ContainerImage {
 	pullSources, err := s.mirrorStore.PullSources(srcImage.GetName().GetFullName())
-	if err != nil {
+	// A not exist error is expected when mirroring is not setup, therefore we do not log it.
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
mentioned above, but we should do `errors.Is(err, fs.ErrNotExist)`, instead. `os.ErrNotExist` is an alias for `fs.ErrNotExist` anyway